### ISSUE TITLE
The Dotenv Component

### ISSUE THREAD

**Issue by VBelozyorov** [symfony/symfony-docs#15166]
Latest version where I can find the Dotenv component's documentation is [4.3](https://symfony.com/doc/4.3/components/dotenv.html). But component itself looks alive on [packagist.org](https://packagist.org/packages/symfony/dotenv).

Was it intended to remove it from docs for 4.4+ or it is some error?

**Comment by javiereguiluz** [comment]
A few months ago we started "removing" the stand-alone component docs. The Symfony Docs focus on features as used inside a Symfony application (here's where we explain env vars -> https://symfony.com/doc/current/configuration.html#configuration-based-on-environment-variables) and the code needed to use the component stand-alone is explained in the README of the component: https://github.com/symfony/dotenv

I know this is not ideal for some people, but our resources are very limited, so we had to make this decision. Thanks for your understanding.
